src/app/layout.tsx

```diff
@@ -1,9 +1,24 @@
 import type { Metadata } from 'next';
-import { Inter } from 'next/font/google';
-import '@/styles/globals.css';
 import Providers from './providers';
+import LocalFont from "next/font/local";
+import '@/styles/globals.css';
 
-const inter = Inter({ subsets: ['latin'] });
+const fontDana = LocalFont({
+  src: [
+    {
+      path: "../assets/fonts/dana/Dana-Regular.woff",
+      weight: "400",
+      style: "normal",
+    },
+    {
+      path: "../assets/fonts/dana/Dana-Bold.woff",
+      weight: "600",
+      style: "normal",
+    },
+  ],
```

Comment:
We can use the variable font. Looks like you only imported static fonts.

Reply:
https://nextjs.org/docs/app/building-your-application/optimizing/fonts#local-fonts